politeiawww/api/v1/v1.go

```diff
@@ -117,6 +117,7 @@ const (
 	ErrorStatusUserNotFound                ErrorStatusT = 27
 	ErrorStatusWrongStatus                 ErrorStatusT = 28
 	ErrorStatusNotLoggedIn                 ErrorStatusT = 29
+	ErrorStatusUserNotPaid                 ErrorStatusT = 30
```

Comment:
This error needs to be added to the map just underneath this code. All errors need a human-readable form.